css/styles.css

```diff
@@ -0,0 +1,27 @@
+
```

Comment:
Check your CSS for whitespace consistency. Between the selector and the `{`, between the `property:` and the `value;` and so on...